azure-pipelines.yml

```diff
@@ -77,6 +75,18 @@ jobs:
         PYTEST_VERSION: '*'
         JOBLIB_VERSION: '*'
         COVERAGE: 'true'
+      pylatest_conda_mkl_no_openmp:
+        DISTRIB: 'conda'
```

Comment:
Since I remove SKLEARN_NO_OPENMP, I added a macos job where we don't set the vars to enable OpenMP. It gives a job without openmp and allows to check the warning.

Reply:
Unfortunately, I think we need at least 2 jobs for macOS: one with openmp enabled and one with openmp disabled.